wagtail/wagtailusers/templates/wagtailusers/groups/list.html

```diff
@@ -4,16 +4,11 @@
         <tr>
             <th class="name">
                 {% trans "Name" %}
-                {% if ordering == "name" %}
-                    <a href="{% url 'wagtailusers_groups:index' %}" class="icon icon-arrow-down-after teal"></a>
-                {% else %}
-                    <a href="{% url 'wagtailusers_groups:index' %}?ordering=name" class="icon icon-arrow-down-after"></a>
-                {% endif %}
             </th>
         </tr>
     </thead>
     <tbody>
-        {% for group in groups %}
+        {% for group in page_obj %}
```

Comment:
Isn't this change unnecessary because of [this line](https://github.com/wagtail/wagtail/pull/3774/files#diff-6136c06de15604ea4f50c092b0aa4718R92) defining the context object name as `groups`?